src/modules/marketplace.ts

```diff
@@ -1124,4 +1124,12 @@ export class MarketplaceModule
       | DirectListing
     )[];
   }
+
+  public async isRestrictedListerRoleOnly(): Promise<boolean> {
+    return this.readOnlyContract.restrictedListerRoleOnly();
+  }
+
+  public async setRestrictedListerRoleOnly(isOpen: boolean): Promise<void> {
```

Comment:
this signature seems wrong - `setRestricted...()` but passes `isOpen` which is the opposite of restricted?

should be `setRestrictedToListerRole(isRestricted: boolean)` (which is what the contract expects)